docker-compose.dev.yml

```diff
@@ -3,12 +3,17 @@ version: '3'
 services:
   meican:
     image: meican-app-dev
+    env_file:
+      - .env
+    entrypoint: ["./docker_for_build/wait-for-mysql.sh", "db", "/docker-entrypoint.sh"]
```

Comment:
I think we can put this part in the docker-compose.yml file as it is important for both dev and prod

Reply:
yes, you are right! I've just sent a change for moving this to `docker-compose.yml` (4f6684cd)